src/actions/index.js

```diff
@@ -0,0 +1,8 @@
+import { select } from '../action_types.js';
```

Comment:
Move action types into this file. They actually should be all cased like SELECT.